.github/workflows/build-and-push-assets.yml

```
@@ -74,7 +78,7 @@ jobs:
       GITHUB_USER_EMAIL: ${{ secrets.GITHUB_USER_EMAIL }}
       GITHUB_USER_NAME: ${{ secrets.GITHUB_USER_NAME }}
       GITHUB_USER_SSH_KEY: ${{ secrets.GITHUB_USER_SSH_KEY }}
-      COMPILE_SCRIPT: ${{ inputs.COMPILE_SCRIPT_DEV }} # we'll override if the push is for tag
+      COMPILE_SCRIPT: ${{ inputs.COMPILE_SCRIPT_DEV }}
```

To reduce confusion, and given this is overwritten later anyway, can we use an empty string as the default value?
```suggestion
      COMPILE_SCRIPT: ''
```